data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,413 @@ mission "Ringworld Debris: Quarg"
+			`	"However, this could easily have been just a coincidence: asteroids of that size need a lot of effort before being able to extract anything useful so maybe they were just combining their forces to get its minerals. Unusual, but far from impossible. So, at this point I had an idea." The screen on the touchpad changes, now showing raw numbers separated on a spreadsheet. "The sensors you carried to Patir had various functions, one of which was a high power asteroid scanner. The asteroid in particular was shown to be extremely rich in a particular metal, iridium. There is more iridium on it than on half a dozen Ember Waste systems, planets included, so the asteroid mining theory was starting to become more and more concrete: however, here's the odd part: the concentration of iridium on the asteroid when you jumped away wasn't lower than when the scanning begun, it was higher! For whatever reason, the Fetri'sei aren't extracting the iridium away of the asteroid, they are depositing it there."`
+			choice
+				`	"Are you sure that it isn't just an error in the measurement?"`
+				`	"Why are they doing that?"`
+					goto why
+			`	"Well, it's a possibility, this instrumentation hasn't been tested in years. But if it isn't, this could give us a lot of insights on the real purpose of these Ka'sei and why they are around that black hole. It's at least worth a try!`
+				goto end
+			label why
+			`	"Well, let's find out! The only way we can find insights on their real purpose is to study this asteroid and what the Ka'sei are doing there.`
+			label end
+			`	"My plan is to take a short range high power scanner, install it on your ship and map the entire asteroid to analyze the site where the Ka'sei are bringing the iridium. Everything happened so fast that I wasn't able to get the scanner I was looking for yet, but if you meet me in the spaceport's cafeteria in a few hours I promise everything will be ready!" Dusk looks so excited that he leaves without leaving you time to ask any question.`
+
+
+mission "Ka'het: Patir Mystery 2"
+	name "Scanners in <system>"
+	description "Make detailed scans of the asteroid Dusk indicated in the <system> system."
+	source "Aventine"
+	destination "Builder Settlement"
+	cargo "short range scanner" 15
+	to offer
+		has "Ka'het: Patir Mystery 1: done"
+	on offer
+		event "asteroid on patir"
+		conversation
+			`As soon as you arrive in the cafeteria you start looking for Dusk, but you can't find him anywhere. He arrives a few minutes later and waves at you. "Sorry if I was late, Captain," he says, " I was finishing to gather the necessary technology for our mission. Everything is ready now."`
+			choice
+				`	"What is the next step?"`
+					goto next
+				`	"Wait, you haven't given me a chance to answer you the last time you met. I decline the offer."`
```

Comment:
It's there because you couldn't decline the previous mission (was an on complete, this is the spaceport one) and the Taely talk about risking remaining stuck there, but I could remove it if needed